### PR TITLE
Add Chinese Translation for 0x90 0x91 & Glossary

### PR DESCRIPTION
**Pull request by SpZanG**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [x] Your PR is linked to an issue.
- [x] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [x] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [x] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR covers issue #340 .

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

File: Document-zhcn/0x90-Appendix-A_Glossary.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- **开放式Web应用程序安全项目(OWASP)** – 开放式Web应用程序安全项目 (OWASP) 是一个全球性的免费开发社区，致力于提高应用程序软件的安全性。我们的任务是使应用程序安全性“可见化”，以便人们和机构能够就应用程序安全风险做出明智的决策。请参阅：<https://www.owasp.org/>
+- **个人验证信息(Personally Identifiable Information (PII))** – PII 是一个可以单独使用或者与其他信息一起使用的信息，用来识别、联络、定位某一个人，或者在上下文中识别个人。
+- **地址无关可执行文件(PIE)** – 地址无关可执行文件(Position-independent executable (PIE))是指可以在存储器中任意位置正确的运行，而不受到绝对地址影响的一种机器码。
+- **公钥基础设施(PKI)** – PKI是一种将公钥与实体的相应身份绑定的协议。绑定是通过认证机构(CA)的注册和发布过程建立的。
+- **静态应用程序安全测试(SAST)** – 静态应用程序安全测试(Static application security testing (SAST))是一系列的技术，其目的是用于分析程序源码，字节码和二进制代码，用于程序代码的编写和陈述安全漏洞的设计条件。SAST解决方案在非运行状态下“由内而外”的分析应用程序。
+- **系统生命周期(SDLC)** – 软件开发的生命周期。
+- **安全架构(Security Architecture)** – 一个程序设计的抽象概念，用于辨识和描述安全控制的使用位置和方式，还可以用于辨识和描述用户和应用程序资料的位置和敏感性。
+- **安全配置(Security Configuration)** – 一个会影响安全控制措施使用方式的应用程序的配置文件。
+- **安全控制措施(Security Control)** – 一个功能或者组件执行安全检查（例如：一个访问控制检查）或者被呼叫时导致的安全结果（例如：生成一个审查记录）。
+- **SQL 注入(SQLi)** – 用于攻击由数据驱动的应用程序，将恶意的SQL语句插入到注入点（entry point）。
```
Comment (jack-smith-dh) [inline Document-zhcn/0x90-Appendix-A_Glossary.md]:
```suggestion
- **SQL注入(SQLi)** – 用于攻击由数据驱动的应用程序，将恶意的SQL语句插入到注入点（entry point）。
```

File: Document-zhcn/0x90-Appendix-A_Glossary.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- **恶意程序(Malware)** – 在程序使用者和管理员不知情的情况下，可执行的程序代码在程序运行的时候被引入到程序。
+- **开放式Web应用程序安全项目(OWASP)** – 开放式Web应用程序安全项目 (OWASP) 是一个全球性的免费开发社区，致力于提高应用程序软件的安全性。我们的任务是使应用程序安全性“可见化”，以便人们和机构能够就应用程序安全风险做出明智的决策。请参阅：<https://www.owasp.org/>
+- **个人验证信息(Personally Identifiable Information (PII))** – PII 是一个可以单独使用或者与其他信息一起使用的信息，用来识别、联络、定位某一个人，或者在上下文中识别个人。
+- **地址无关可执行文件(PIE)** – 地址无关可执行文件(Position-independent executable (PIE))是指可以在存储器中任意位置正确的运行，而不受到绝对地址影响的一种机器码。
+- **公钥基础设施(PKI)** – PKI是一种将公钥与实体的相应身份绑定的协议。绑定是通过认证机构(CA)的注册和发布过程建立的。
+- **静态应用程序安全测试(SAST)** – 静态应用程序安全测试(Static application security testing (SAST))是一系列的技术，其目的是用于分析程序源码，字节码和二进制代码，用于程序代码的编写和陈述安全漏洞的设计条件。SAST解决方案在非运行状态下“由内而外”的分析应用程序。
+- **系统生命周期(SDLC)** – 软件开发的生命周期。
+- **安全架构(Security Architecture)** – 一个程序设计的抽象概念，用于辨识和描述安全控制的使用位置和方式，还可以用于辨识和描述用户和应用程序资料的位置和敏感性。
+- **安全配置(Security Configuration)** – 一个会影响安全控制措施使用方式的应用程序的配置文件。
+- **安全控制措施(Security Control)** – 一个功能或者组件执行安全检查（例如：一个访问控制检查）或者被呼叫时导致的安全结果（例如：生成一个审查记录）。
```
Comment (jack-smith-dh) [inline Document-zhcn/0x90-Appendix-A_Glossary.md]:
```suggestion
- **安全控制措施(Security Control)** – 一个功能或者组件执行安全检查（例如：一个访问控制检查）或者被调用时导致的安全结果（例如：生成一个审查记录）。
```

File: Document-zhcn/GLOSSARY.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+一种软件测试方法，在不知道内部结构和运作方式的状况下检测应用程序的功能。
+
+## 组件(Component)
+
+一个独立的代码单元，具有与其他组件交流的关联磁盘和网络界面。
+
+## 跨站脚本(XSS)
+
+一个常见的网站安全漏洞，它准许客户端的脚本注入到网站的内容里。
```
Comment (jack-smith-dh) [inline Document-zhcn/GLOSSARY.md]:
```suggestion
一个常见的网站安全漏洞，它导致受到攻击者控制的客户端脚本注入到网站的内容里。
```

File: Document-zhcn/GLOSSARY.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+用于加密算法和/或生成加密密钥的软件，硬件或/和固件。
+
+## 通用缺陷列表(CWE)
+
+CWE是由社区开发的常见安全漏洞列表。它是一种通用的语言，是软件安全工具的衡量标准也是漏洞识别，缓解以及预防工作的基准。
+
+## 动态应用程序安全测试(DAST)
+
+动态应用程序安全测试 (DAST) 技术被设计用来检测处于运行状态的应用程序的安全漏洞。
```
Comment (jack-smith-dh) [inline Document-zhcn/GLOSSARY.md]:
```suggestion
动态应用程序安全测试 (DAST) 技术被设计于用来检测处于运行状态的应用程序的安全漏洞。
```

File: Document-zhcn/GLOSSARY.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+
+存储在设备本身之中的加密密钥。
+
+## 进程间通信(IPC)
+
+进程间通信 （Inter Process Communications）,在IPC，处理程序之间的通信交流并且与内核进行通信交流以协调其活动。
+
+## 输入验证(Input Validation)
+
+一个规范化和验证用来处理不受信任用户的输入。
```
Comment (jack-smith-dh) [inline Document-zhcn/GLOSSARY.md]:
```suggestion
一个规范化和验证用来处理不受信任用户的输入的机制。
```

File: Document-zhcn/GLOSSARY.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+
+## 开放式Web应用程序安全项目(OWASP) 
+
+开放式Web应用程序安全项目 (OWASP) 是一个全球性的免费开发社区，致力于提高应用程序软件的安全性。我们的任务是使应用程序安全性“可见化”，以便人们和机构能够就应用程序安全风险做出明智的决策。请参阅： <https://www.owasp.org/>
+
+## 个人验证信息(Personally Identifiable Information (PII)) 
+
+PII 是一个可以单独使用或者与其他信息一起使用的信息，用来识别、联络、定位某一个人，或者在上下文中识别个人
+
+## 地址无关可执行文件(PIE)
```
Comment (jack-smith-dh) [inline Document-zhcn/GLOSSARY.md]:
```suggestion
## 不基于地址的可执行代码(PIE)
```

File: Document-zhcn/GLOSSARY.md [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+
+## 公钥基础设施(PKI) 
+
+PKI是一种将公钥与实体的相应身份绑定的协议。绑定是通过认证机构(CA)的注册和发布过程建立的。
+
+## 静态应用程序安全测试(SAST) 
+
+静态应用程序安全测试(Static application security testing (SAST))是一系列的技术，其目的是用于分析程序源码，字节码和二进制代码，用于程序代码的编写和陈述安全漏洞的设计条件。SAST解决方案在非运行状态下“由内而外”的分析应用程序。
+
+## 系统生命周期(SDLC)
```
Comment (jack-smith-dh) [inline Document-zhcn/GLOSSARY.md]:
```suggestion
## SDLC
```

File: Document-zhcn/GLOSSARY.md [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+
+一个程序设计的抽象概念，用于辨识和描述安全控制的使用位置和方式，还可以用于辨识和描述用户和应用程序资料的位置和敏感性。
+
+## 安全配置(Security Configuration)
+
+一个会影响安全控制措施使用方式的应用程序的配置文件。
+
+## 安全控制措施(Security Control) 
+
+一个功能或者组件执行安全检查（例如：一个访问控制检查）或者被呼叫时导致的安全结果（例如：生成一个审查记录）。
```
Comment (jack-smith-dh) [inline Document-zhcn/GLOSSARY.md]:
```suggestion
一个功能或者组件执行安全检查（例如：一个访问控制检查）或者被调用时导致的安全结果（例如：生成一个审查记录）。
```

File: Document-zhcn/GLOSSARY.md [hunk 9]
```diff
@@ -0,0 +1,161 @@
...
+
+## SQL 注入(SQLi)
```
Comment (jack-smith-dh) [inline Document-zhcn/GLOSSARY.md]:
```suggestion
## SQL注入(SQLi) 
```

File: Document-zhcn/GLOSSARY.md [hunk 10]
```diff
@@ -0,0 +1,161 @@
...
+
+一个功能或者组件执行安全检查（例如：一个访问控制检查）或者被呼叫时导致的安全结果（例如：生成一个审查记录）。
+
+## SQL 注入(SQLi) 
+
+用于攻击由数据驱动的应用程序，将恶意的SQL语句插入到注入点（entry point）。
+
+## 单点登录身份验证(SSO Authentication) 
+
+单点登录身份验证(Single Sign On(SSO))发生在当用户登录到一个客户端然后自动登录到其他的客户端，不论用户使用任何平台，技术或者是域名。例如：当你在登录google时候，会自动登录到youtube，google文档和google邮件服务。
```
Comment (jack-smith-dh) [inline Document-zhcn/GLOSSARY.md]:
```suggestion
不论用户使用任何平台，技术或者是域名，单点登录身份验证(Single Sign On(SSO))可以让用户登录到一个客户端然后自动登录到其他的客户端。例如：当一个用户在登录谷歌时候，会自动登录到油管，谷歌文档和Gmail。
```

File: Document-zhcn/GLOSSARY.md [hunk 11]
```diff
@@ -0,0 +1,161 @@
...
+
+## 威胁模型分析(Threat Modeling) 
+
+一种技术。该技术包括开发日益完善的安全体系结构以识别威胁媒介，安全区域，安全控制以及重要的技术和业务资产。
```
Comment (jack-smith-dh) [inline Document-zhcn/GLOSSARY.md]:
```suggestion
一种开发日益完善的安全体系结构以识别威胁媒介，安全区域，安全控制以及重要的技术和业务资产的方法。
```

File: Document-zhcn/GLOSSARY.md [hunk 12]
```diff
@@ -0,0 +1,161 @@
...
+
+统一资源标志符是用于识别名称或者网络资源的字符串。统一资源定位器通常用作资源的引用。
+
+## 用户验收测试(User acceptance testing (UAT)) 
+
+所有的软件在上线前会在一个跟生产环境相似的测试环境中运行。
+
+## 验证者(Verifier)
+
+根据OWASP MASVS的要求审核应用程序的个人或者团队。
```
Comment (jack-smith-dh) [inline Document-zhcn/GLOSSARY.md]:
```suggestion
基于OWASP MASVS要求，审核应用程序的个人或者团队。
```

File: Document-zhcn/0x90-Appendix-A_Glossary.md [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+
+- **地址空间配置随机加载 (ASLR)** – 一种能让内存损坏漏洞的攻击更加困难的技术。
+- **应用程序安全(Application Security)** – 应用程序安全是针对OSI模型应用组件的分析，而不是针对操作系统或者网络连接。
+- **应用程序安全验证(Application Security Verification)** – 针对应用程序使用 OWASP MASVS 进行技术评估。
+- **应用程序安全验证报告(Application Security Verification Report)** – 一份针对特定的应用程序，由检测者制作并记录所有结果与相关分析的报告。
+- **身份验证(Authentication)** – 对应用程序的使用者身份验证。
+- **自动化验证(Automated Verificatio)** – 使用自动化工具（包括动态分析，静态分析或者两者皆使用），利用漏洞特征来挖掘漏洞。
+- **黑盒测试(Black box testing)** – 一种软件测试方法，在不知道内部结构和运作方式的状况下检测应用程序的功能。
+- **组件(Component)** – 一个独立的代码单元，具有与其他组件交流的关联磁盘和网络界面。
+- **跨站脚本(XSS)** – 一个常见的网站安全漏洞，它导致受到攻击者控制的客户端脚本注入到网站的内容里。
```
Comment (SpZanG) [inline Document-zhcn/0x90-Appendix-A_Glossary.md]:
```suggestion
- **跨站脚本(XSS)** – 一个常见的网站安全漏洞，它导致受到由攻击者控制的客户端脚本注入到网站的内容里。
```

**Comment by jack-smith-dh** [comment]
reviewed, good to proceed

**Pull request merged**